spec/features/applications/application_show_spec.rb

```diff
@@ -76,19 +76,19 @@
     click_button "Submit Application"
 
     visit "/pet_applications/#{PetApplication.last.id}"
+    expect(current_path).to eq("/pet_applications/#{PetApplication.last.id}")
 
     expect(page).to have_content("12345 Jesse Ave")
     expect(page).to have_content("Jesse, CO 80120")
     expect(page).to have_content("303-867-5309")
     expect(page).to have_content("Because I'm too cool for school")
 
     within "#app-#{PetApplication.last.id}-pet-#{pet_1.id}" do
-      expect(page).to have_link("#{pet_1.name}", href:"/pets/#{pet_1.id}")
+      expect(page).to have_link("Approve #{pet_1.name}'s Application")
```

Comment:
Nice personalization here with the pets name!  Not sure if that was part of the story but I like it!